hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/MemberStateImpl.java

```diff
@@ -302,6 +312,8 @@ public JsonObject toJson() {
         serializeAsMap(root, "reliableTopicStats", reliableTopicsWithStats);
         serializeAsMap(root, "pnCounterStats", pnCountersWithStats);
         serializeAsMap(root, "executorStats", executorsWithStats);
+        serializeAsMap(root, "scheduledExecutorsWithStats", scheduledExecutorsWithStats);
```

Comment:
Minor: Would it be better to rename those as `scheduledExecutorStats` and `durableExecutorStats` to be consistent with others?

Reply:
done.